auth/simple.go

```diff
@@ -70,3 +117,40 @@ func (a simpleAuth) GetUser(ctx context.Context) (string, error) {
 	}
 	return id.(string), nil
 }
+
+func (a simpleAuthN) authenticate(userID string) error {
+	if userID == "" {
+		return fmt.Errorf("Must provide header: [%s]", a.userIDHeader)
+	}
+	return nil
+}
+
+func (a simpleAuthZ) authorize(httpMethod, userID string) error {
+	switch httpMethod {
+	case "GET", "":
+		return a.authorizeUser(a.whitelistReads, a.readWhitelistedUserIDs, userID)
+	case "POST", "PUT", "PATCH", "DELETE":
```

Comment:
When will the method be `PATCH`?

Reply:
I found a route in service.go (~ln 357):
`
router.Handle(mappingRuleWithIDPath, h.wrap(s.updateMappingRule)).Methods(http.MethodPut, http.MethodPatch)
`